packages/react/src/patterns/sub-patterns/ContentGroup/README.md

```diff
@@ -70,6 +70,7 @@ Add the following line on your `.env` file at the root of your project,
 | Name                           | Description                   |
 | ------------------------------ | ----------------------------- |
 | `dds--content-group`           | Content group wrapper element |
+| `dds--content-group_cta`       | Content group CTA element     |
```

Comment:
@kennylam it looks weird that this is one underscore instead of two...

Reply:
oops... typo let me fix that.